spinalcordtoolbox/utils.py

```diff
@@ -149,6 +149,24 @@ def flush(self):
         self.fd2.flush()
 
 
+def abspath(fname):
```

Comment:
I don't think tilde expansion should be done anywhere else than in the user's shell.

Reply:
the problem occurs when users enter tilde in config files (e.g. [here](https://github.com/neuropoly/spinalcordtoolbox/blob/master/scripts/sct_run_batch.py#L59)). Even if users are not supposed to do it, they will do it.

What is the cons for dealing with the tilde in this situation?